src/robot/client.cpp

```diff
@@ -1,5 +1,7 @@
 #include <algorithm>
```

Comment:
largely just enforcing cpp/hpp distinction and defining a couple trivial functions. not particularly important for review.